File: patterns/javascript-patterns/pattern95.js
// Pattern 95: ...

//     *        *
//    ***      ***
//   *****    *****
//  *******  *******
// ******************

let minStars = 1;
let pHeight = 5;
let pSpace = pHeight - 1;
let n;
let x = 1;

for (let i = 0; i < pHeight; i++) {
    let row = '';

    for (let j = pSpace; j > i; j--) {
        row += ' ';
    }

    for (let k = 0; k < minStars; k++) {
        row += '*';
    }

    for (n = pHeight + pHeight - 2; n >= x; n--) {
        row += ' ';
    }

    for (let k = 0; k < minStars; k++) {
        row += '*';
    }

    minStars += 2;
    x += 2;
    console.log(row);
}

File: patterns/python-patterns/pattern95.py
# Pattern 95: ...

#     *        *
#    ***      ***
#   *****    *****
#  *******  *******
# ******************

min_stars = 1

p_height = 5
p_space = p_height - 1

for i in range(p_height):
    for j in range(p_space, i, -1):
        print(" ", end="")
    for k in range(min_stars):
        print("*", end="")
    for l in range(p_space * 2, p_space - 1, -1):
        print(" ", end="")
    for m in range(min_stars):
        print("*", end="")
    
    min_stars += 2
    print()